scripts/zowe-prepare-runtime.sh

```diff
@@ -59,3 +59,10 @@ cp -R ./explorer-MVS $ZOWE_ROOT_DIR/
 cp com.ibm.atlas.explorer-MVS.json $ZOWE_ROOT_DIR/$zluxserverdirectory/plugins/
 
 chtag -b $ZOWE_ROOT_DIR/explorer-MVS/web/images/explorer-MVS.png
+
+# Add plugins to pinned plugins
+pinnedPluginsDirectory=$ZOWE_ROOT_DIR/$zluxserverdirectory/deploy/product/ZLUX/pluginStorage/com.rs.mvd.ng2desktop/ui/launchbar/plugins
+chmod u+w $pinnedPluginsDirectory/pinnedPlugins.json
+awk '/"com.rs.mvd.vt",/{print "\"com.ibm.explorer-jes\","}1' $pinnedPluginsDirectory/pinnedPlugins.json > $TEMP_DIR/tempPinned
+awk '/"com.rs.mvd.vt",/{print "\"com.ibm.explorer-mvs\","}1' $TEMP_DIR/tempPinned > $TEMP_DIR/tempPinned2
+awk '/"com.rs.mvd.vt",/{print "\"com.ibm.explorer-uss\","}1' $TEMP_DIR/tempPinned2 > $pinnedPluginsDirectory/pinnedPlugins.json
```

Comment:
This has become out of date as com.rs.mvd.vt was renamed to org.zowe.terminal.vt